ci/pipeline-backup-metadata.yml

```diff
@@ -10,43 +10,26 @@ resource_types:
   source:
       repository: cftoolsmiths/toolsmiths-envs-resource
 
+- name: terraform
+  type: docker-image
+  source:
+    repository: ljfranklin/terraform-resource
+
 resources:
-- name: ci
+- name: capi-ci
   type: git
   icon: github-box
   source:
+    branch: update-backup-metadata-pipeline
```

Comment:
I have this pipeline pointed at the branch for testing it. I'll switch this back before merging.

Reply:
https://github.com/cloudfoundry/capi-ci/pull/14/commits/d7727f203fa3e5f1d3421f5cc26d7dc2c35d039c